gems/smithy/lib/smithy/templates/client/auth_plugin.erb

```diff
@@ -83,29 +83,22 @@ module <%= module_name %>
             return
           end
 
-          identity_properties = auth_option.identity_properties
-          identity = identity_provider.identity(identity_properties)
+          identity = identity_provider.identity({})
 
           ResolvedAuth.new(
-            scheme_id: scheme_id,
```

Comment:
I think we still need scheme_id here right? Otherwise we are not sure what auth we actually resolved.